.github/workflows/udd.yml

```diff
@@ -0,0 +1,53 @@
+name: udd-update-dependencies
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "42 8 * * *"
+
+jobs:
+  update-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: denoland/setup-deno@v1
+        with:
+          deno-version: 1.15.3
+      - name: Update dependencies
+        run: |
+          deno run -A https://deno.land/x/udd/main.ts deps.ts test_deps.ts
+      - name: Create Pull Request
```

Comment:
Could we change that instead failing most of the time it will just skip all the steps and don't fail the action?

Reply:
I just copied this code from udd repo. it should works well